### ISSUE TITLE
[Unit 4] "Standards and Guidelines" not different things

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-curricula#97]
The wording "standards and guidelines" is a difficult issue. They are all "standards". WCAG, ATAG, and UAAG happen to have "guidelines" in their title. ARIA does not. (And to further complicate it, UAAG 1.0 is technically a standard; whereas UAAG 2.0 is not, it's just a Note.) 

However, "standards and guidelines" makes it seem like they are 2 different things. I feel fairly strongly about avoiding that phrasing.

Note in the WAI site navigation, we used "Standards/Guidelines" (with a slash, not "and"). And that page < https://www.w3.org/WAI/standards-guidelines/> summary is: "This page introduces guidelines **and other standards** related to web accessibility." (since some people will be looking for "guidelines") We very deliberately used "Guidelines" and "Standards" in the specific headings and links. For example, h2 "Accessibility Guidelines" to clarify different from h2 "Technical Specifications". And the links include "standard", e.g., "WCAG 2.1 Standard".

In my pull request with ideas ([preview 96](https://deploy-preview-96--wai-curricula.netlify.com/curricula/1-4/)), I changed some wording to "guidelines and other standards" – however, I didn't change all the titles or change it in all places…

Maybe the Unit Title should be something like

> Principles, Guidelines, and other Standards in Practice

**Comment by daniel-montalvo** [comment]
Good point, I totally missed that, and always worked on this thinking of guidelines and standards from a hierarchical point of view, that is, principles > standards > guidelines.

Will work on this as well.
